workflows/VGP-assembly-v2/VGP-meryldb-creation-trio/meryldb-creation-trio.ga

```diff
@@ -0,0 +1,1249 @@
+{
```

Comment:
This workflow doesn't have a README.md file, is that intentional ?
I guess we should at least mention how this workflow differs from meryldb-creation ?

Reply:
Yes, I'll include it. Thanks!